platform/admin/ui/src/main/webapp/js/views/configuration/ConfigurationEdit.view.js

```diff
@@ -28,9 +29,10 @@ define([
     'text!templates/configuration/configurationItem.handlebars',
     'text!templates/configuration/textTypeListHeader.handlebars',
     'text!templates/configuration/textTypeList.handlebars'
-], function (Marionette, ich, _, Backbone, wreqr, Spinner, $, AlertsModel, AlertsView, configurationEdit, configurationItem, textTypeListHeader, textTypeList) {
+], function (Marionette, ich, _, Backbone, wreqr, Spinner, spinnerConfig, $, AlertsModel, AlertsView, configurationEdit, configurationItem, textTypeListHeader, textTypeList) {
 
     var ConfigurationEditView = {};
+    spinnerConfig.color = '#000000';
```

Comment:
Ah, I wasn't sure if the clone was necessary, or if each `require` was a separate instance.